assets/opstools/AppBuilder/classes/rules/ABViewGridFilterMenu.js

```diff
@@ -98,7 +100,21 @@ export default class ABViewGridFilterMenu {
 					},
 
 					{
-						view: "segmented",
+						view: "radio",
+						id: ids.filterGlobal,
+						hidden: true,
+						vertical: true,
+						label: "Show",
+						labelWidth: App.config.labelWidthLarge,
+						options: [
+							{ id: "default", value: "All matching records" },
+							{ id: "single", value: "Single records only"}
```

Comment:
So what do the 2 options represent?

Reply:
Ric was interested in the search to hide results unless a single record is matched. So by default it will work as you would expect...there is a search above the grid and as you type it filters content. The second option will hide the grid below and as you type if a single record matches it will show.